app/src/views/search-engine/index.js

```diff
@@ -44,7 +47,48 @@ const SearchEngine = () => {
         setLoading={setLoading}
         loading={loading}
       />
+      <Chip
+        size='small'
+        label={`${articles.length} resultados`}
+        className={classes.resultsChip}
+      />
       <ArticlesList articles={articles} />
+      {articles.length === 0 &&
+        <Card className={classes.card}>
+          <CardContent>
+            <Typography gutterBottom variant='h5' component='h2'>
+                Motor de búsqueda - Reglamento Vial de Monterrey
+            </Typography>
+            <Typography
+              variant='body1'
+              color='textSecondary'
+              component='p'
+              className={classes.cardContent}
+            >
+                Algoritmo para realizar búsquedas optimizadas sobre los <b>reglamentos viales oficiales</b> de tu comunidad.
```

Comment:
I'd like an english card as well to make it easier for the hiring committe or any english only reader to understand

Reply:
I wouldn't worry too much about this. For users who don't speak Spanish, Chrome will automatically offer to translate the page for them.

Reply:
I actually don't think it'd be a bad idea unless it'd take a long time. Or at least to make an hideable English version that users can click on. A good portion of your audience, including the people you present to in the final STEP presentation will be primarily English speaking. 

If you choose not to include an English card, you'll want to double check all the text translates well with Google translate.

(Also, what if someone uses not-Chrome? *horrified gasp*)